src/parser/monk/windwalker/modules/resources/ChiDetails.js

```diff
@@ -18,6 +20,10 @@ class ChiDetails extends Analyzer {
     return this.chiTracker.wasted;
   }
 
+  get chiWastedPercent() {
+    return this.chiWasted / (this.chiWasted + this.chiTracker.generated);
```

Comment:
very unlikely but it might crash
please add a default value

```suggestion
return (this.chiWasted / (this.chiWasted + this.chiTracker.generated)) || 0
```